### DISCUSSION TITLE
Project64 with no-intro titles

### DISCUSSION THREAD

**Discussion by leov30** [project64/project64#2237] [Show and tell]
hi there,
I used a script to replace the "Good Name" titles for the "No-intro 04/2022" in this files: Audio.rdb, Project64.rdb, Project64.rdx, and Video.rdb. I also included a list of No-intro games that were missing in Project64.rdx and Project64.rdb.
I skip this games and all wii virtual console and iQue ROMs, because of nul or dup CRC header:

```
Blues Brothers 2000 (Europe) (En,Fr,De,Es,It,Nl) (Beta) (2000-01-15) (#53)
Mortal Kombat Trilogy (USA) (Beta) (1996-05-13)
GameBooster 64 (USA) (v1.1) (Unl)	
Turok - Dinosaur Hunter (Europe) (Rev 1)
Turok - Dinosaur Hunter (Germany) (Rev 2)
Turok - Dinosaur Hunter (USA) (Rev 1)
Turok - Dinosaur Hunter (USA) (Rev 2)
Nintendo 64 Modem (Japan) (Add-on)
```

[project64_nointro_titles.zip](https://github.com/project64/project64/files/8790783/project64_nointro_titles.zip)

**Comment by DerekTurtleRoe** [comment]
@leov30 At some point we are going to replace all Good Names with No-Intro names, but I have to figure out how best to tackle that. I've already replaced some of them, but I do appreciate this post. I will look into making some of these changes soon.
